frontend-react/src/components/header/ReportStreamHeader.tsx

```diff
@@ -79,14 +79,215 @@ interface ReportStreamNavbarProps extends PropsWithChildren {
     onToggleMobileNav: () => void;
     isMobileNavOpen: boolean;
     user: RSSessionContext["user"];
+    activeMembership: MembershipSettings | null | undefined;
+    logout: () => void;
     containerRef: MutableRefObject<HTMLElement | null>;
 }
 
-function ReportStreamNavbar({
+function ReportStreamAuthNavbar({
     children,
```

Comment:
Since this component is getting pretty large, might be worth separating this `ReportStreamAuthNavbar` into a different file.

Reply:
I don't think if its worth separating the  ReportStreamAuthNavbar into its own file since it shares a lot of the same dependencies as the ReportStreamNavbar.